src/apps/common/submodules/portListing/portListing.js

```diff
@@ -999,6 +1000,22 @@ define(function(require) {
 			});
 		},
 
+		portListingRequestByAgent: function(args) {
```

Comment:
Try to keep methods with `Request` in their name for methods containing actual request calls (e.i. with either `callApi()` or `monster.request()`).